xml/System.IO.Compression/BrotliStream.xml

```diff
@@ -843,14 +924,26 @@
         <Parameter Name="cancellationToken" Type="System.Threading.CancellationToken" />
       </Parameters>
       <Docs>
-        <param name="buffer">To be added.</param>
-        <param name="offset">To be added.</param>
-        <param name="count">To be added.</param>
-        <param name="cancellationToken">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="buffer">The buffer that contains the data to compress.</param>
+        <param name="offset">The zero-based byte offset in <paramref name="buffer" /> from which to begin copying bytes to the Brotli stream.</param>
+        <param name="count">The maximum number of bytes to write.</param>
+        <param name="cancellationToken">The token to monitor for cancellation requests. The default value is <see cref="P:System.Threading.CancellationToken.None" />.</param>
+        <summary>Asynchronously writes compressed bytes to the underlying Brotli stream from the specified byte array.</summary>
```

Comment:
We are missing exception details. For example:
https://github.com/dotnet/corefx/blob/306fe9d60cc6db02a69d8ea6b88dde2b98fe8a7c/src/System.IO.Compression.Brotli/src/System/IO/Compression/enc/BrotliStream.Compress.cs#L75